screens/HomeScreen.js

```diff
@@ -38,11 +39,72 @@ export default class HomeScreen extends React.Component {
     header: null
   };
 
-  selectLandmark = data => {
+  fetchLandmarkPlaceId(address) {
```

Comment:
can we delete this function since we're going to be storing the placeIds? related question -- is it possible to get the placeIds some other way than through code? if not, maybe we can explore writing a script that will cycle through the landmarks, fetch the place ids, and push the back up to firebase and we can run this script when we add new landmarks. what do you think?

Reply:
You can apparently get the placeIds from the [Google Place Id Finder](https://google-developers.appspot.com/maps/documentation/javascript/examples/full/places-placeid-finder) as well. 

But if the script isn't too much trouble that might be more efficient than manually checking every address. I guess it also depends on how much data we have in the end.